ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -608,6 +674,50 @@ def get_connections(
             session_type, pin_or_relay_names, sites, instrument_type_id
         )
 
+    def get_connections_with_multiplexer(
+        self,
+        session_type: Type[TSession],
+        multiplexer_session_type: Type[TMultiplexerSession],
+        pin_or_relay_names: Union[str, Iterable[str], None] = None,
+        sites: Union[int, Iterable[int], None] = None,
+        instrument_type_id: Optional[str] = None,
+    ) -> Sequence[TypedConnectionWithMultiplexer[TSession, TMultiplexerSession]]:
+        """Get all connections matching the specified criteria.
+
+        This is a generic method that supports any instrument driver.
+
+        Args:
+            session_type: The instrument session type.
+
+            multiplexer_session_type: The multiplexer(s) session type.
```

Comment:
"multiplexer" does not have to be plural to modify "session type".

```suggestion
            multiplexer_session_type: The multiplexer session type.
```

Replace this everywhere.

Reply:
Sure.